lib/OnyxUtils.ts

```diff
@@ -742,8 +670,8 @@ function keyChanged(key, data, prevData, canUpdateSubscriber = () => true, notif
             // returned by the selector and only if the selected data has changed.
             if (subscriber.selector) {
                 subscriber.withOnyxInstance.setStateProxy(() => {
-                    const previousValue = getSubsetOfData(prevData, subscriber.selector, subscriber.withOnyxInstance.state);
```

Comment:
I'm starting to wonder if removing `getSubsetOfData` was a good idea, it was encapsulating this logic of using the selector to get the data and for know it was just calling the selector, but in the future if we want to change or make it more complex we will have to replicate it everywhere now. wdyt?

Reply:
Well I don't agree but it's up to you to keep this way, it's also ok for me 🙂